sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVInferSchemaSuite.scala

```diff
@@ -68,4 +68,10 @@ class InferSchemaSuite extends SparkFunSuite {
     assert(CSVInferSchema.inferField(DoubleType, "\\N", "\\N") == DoubleType)
     assert(CSVInferSchema.inferField(TimestampType, "\\N", "\\N") == TimestampType)
   }
+
+  test("Merging Nulltypes should yeild Nulltype.") {
+    assert(
+      CSVInferSchema.mergeRowTypes(Array(NullType),
```

Comment:
Maybe a `val` for this `CSVInferSchema.mergeRowTypes(Array(NullType),Array(NullType))`?